.github/workflows/validations.yaml

```diff
@@ -36,8 +36,8 @@ jobs:
       - name: Restore Java test-fixture cache
         uses: actions/cache@v3
         with:
-          path: syft/pkg/cataloger/java/test-fixtures/java-builds/packages
-          key: ${{ runner.os }}-unit-java-cache-${{ hashFiles( 'syft/pkg/cataloger/java/test-fixtures/java-builds/packages.fingerprint' ) }}
+          path: syft/pkg/cataloger/java/test-fixtures/java-builds
```

Comment:
this cache would overwrite what's checked into git (all other locations under java-builds that is outside of packages)

Reply:
^ My fault - I thought this is what we wanted where we were checking for any changes under `java-builds` and rebuilding on changes to those files - thinking more segmenting out the packages folder is the right choice since it keeps the cache smaller, we probably just want to update the makefile to also key off of rebuilding if there are changes to any of the shell scripts `*.sh`